cosmos/core/cmd_fxn/signature.py

```diff
@@ -30,7 +30,16 @@ def gen_params():
 import decorator
 
 
-def default_prepend(task):
+def default_prepend(task):  # pylint: disable=unused-argument
+    """
+    Set common error- and signal-handling behavior for Cosmos Tasks.
+
+    With these options, Tasks that run multiple commands will error out at the
+    first sign of failure, even if the failure occurs in a multiple-step pipe.
+
```

Comment:
I don't follow the comment "With these options, Tasks that run multiple commands will error out at the
  first sign of failure, even if the failure occurs in a multiple-step pipe."  is that for --fail-fast?